src/components/ui/switch.tsx

```diff
@@ -0,0 +1,29 @@
+"use client"
+
+import * as React from "react"
+import * as SwitchPrimitives from "@radix-ui/react-switch"
+
+import { cn } from "@/lib/utils"
+
+const Switch = React.forwardRef<
```

Comment:
This doesn't seem to be used anymore, and can probably be cleaned up.

There are a bunch of ui components like these, and they seem quite verbose. Did you create them, or copy them from somewhere? I'm just a bit surprised that all this is needed, but then again, I'm mainly a backend dev 😛